package.json

```diff
@@ -15,7 +15,7 @@
     "semantic-release": "semantic-release"
   },
   "engines": {
-    "node": ">=10",
+    "node": ">=12",
```

Comment:
Same as with TypeScript: Not sure if this is gonna work in all future versions of Node.js